lib/iris/tests/test_quickplot.py

```diff
@@ -149,6 +149,7 @@ def test_contourf(self):
     def test_contourf_nameless(self):
         cube = self._small()
         cube.standard_name = None
+        cube.attributes['STASH'] = ''
```

Comment:
``The file header at lib/iris/tests/test_quickplot.py is out of date. The last commit was in 2018, but the copyright states it was 2016.``

😢 

I'm having a robust discussion with our legal department to get rid of the year in the copyright. Watch this space... 🙏 📖 👮

Reply:
Haha, that would be great - it gets me every time! At some point I'll setup PyCharm to run the style check locally...